src/test/ui/issues/issue-43988.rs

```diff
@@ -11,16 +11,6 @@ fn main() {
     let _b = 4;
     //~^^ ERROR attribute should be applied to function or closure
 
-    #[repr(nothing)]
```

Comment:
why did you remove these 2 reprs?

Reply:
whoops! I think that may have been a leftover of me trying to figure out something, let me try adding them back here.